### PR TITLE
remove GCC7 hint

### PR DESCRIPTION
**Pull request by bgruening**
@bioconda/core I think this note can be removed now?

:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
Yup, we should put in the change about pinging core to get added. Shall I put that in your PR or make a separate one?

**Comment by bgruening** [comment]
Feel free to take this over!

**Comment by dpryan79** [comment]
There's a first stab at things, it'd be nice if someone looked it over. I'll mention this in the "Contributing" section of the documentation.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :information_source:
-Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).
+If this is your first pull request to Bioconda please ping @bioconda/core so it can be reviewed and to request being added as a member of the bioconda team.
```
Comment (bgruening) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'm wondering if this note will now ping us every time :)

Maybe we should explain what it means to be added to the bioconda-team? We can also link to docs I guess.

Comment (dpryan79) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, I'm not sure if we'll get auto-pinged or not. If so we have to change that. Yes, maybe we should update the docs first.

**Comment by bgruening** [comment]
Danke!

**Pull request merged**